### DISCUSSION TITLE
Workers resiliency

### DISCUSSION THREAD

**Discussion by B-tronics** [Netflix/conductor#3094] [Q&A]
Hello all!

I am just curious what happens, if a worker somehow fails to complete it's task. Is there a mechanism built in, so that another worker could take over the task from where the previous worker stopped? Is there any way to know what state the worker currently have?

**Comment by dougsillars** [comment]
There is currently no way to know the state of a worker with Conductor. However, assuming you have multiple workers polling for the same task in Conductor, you can set timeouts and retries for the tasks. So if a worker does not respond "in time", you can retry the task - and another worker will pick up the task to complete it.
